images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -240,6 +240,17 @@ function Get-CachedDockerImages {
     return $images
 }
 
+function Get-CachedDockerImagesTableData {
```

Comment:
Why don't you use json format?

```
docker images --digests --format='{{json .}}' | ConvertFrom-Json | Select-Object @{n="Repository:Tag";e={"$($_.Repository):$($_.Tag)"}},Digest,@{n="Created" ; e = {$_.CreatedAt.Split()[0]}}

Repository:Tag Digest                                                                  Created
-------------- ------                                                                  -------
ubuntu:latest  sha256:fff16eea1a8ae92867721d90c59a75652ea66d29c05294e6e2f898704bdb8cf1 2020-10-23
alpine:latest  sha256:185518070891758909c9f839cf4ca393ee977ac378609f700f60a771a2dfe321 2020-05-30
```

Reply:
It is inherited code, i guess for sake of normalisation: stripping headers and having named fields. It is looking nicer in powershell context indeed over sh-like transformations